x-pack/plugins/lens/public/editor_frame_plugin/editor_frame/editor_frame.test.tsx

```diff
@@ -439,4 +421,284 @@ describe('editor_frame', () => {
       );
     });
   });
+
+  describe('suggestions', () => {
+    it('should fetch suggestions of currently active datasource', async () => {
```

Comment:
This says that it's fetching suggestions on first render, without user interaction. Why would we want that?

Reply:
This test loads the editor with an initial datasource and an initial visualization. This means the datasource will be able to provide suggestions for datatables and all visualizations can particiapte in creating suggestions. This is done to fill the bottom right suggestions:
![Untitled drawing](https://user-images.githubusercontent.com/1508364/58313169-39265980-7e0d-11e9-8809-ecfc9ae2fd96.jpg)